doc/Cluster_Setup.md

```diff
@@ -341,11 +341,11 @@ Create a virtual machine.
 ### 3.2. Generate SSH keys
 
 Follow
-[these steps](CortxMotrQuickStart.md#Accessing-the-code-right-way).
+[these steps](https://github.com/Seagate/cortx/blob/main/doc/SSH_Public_Key.rst).
```

Comment:
@yanqingfu , this URL should be relative since it is in the same repo.  Can you change it please?

Reply:
@yanqingfu , I'm gonna go ahead and merge this but you never actually resolved this.  You have the link being:

[https://github.com/Seagate/cortx/blob/main/doc/SSH_Public_Key.rst](https://github.com/Seagate/cortx/blob/main/doc/SSH_Public_Key.rst)

Which is an absolute link.  It should be a relative link like:

SSH\_Public\_Key.rst

Can you please do another PR to fix this?